### PR TITLE
21908 - Updated variables for demonstration

### PR DESCRIPTION
**Pull request by kevashans**
Adjusted GPIO pins for wire management
image

### CODE REVIEW

File: sensors_script/device_availability_monitor.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def get_log_tail():
     read the last 10 lines of upflux and dpkg log file 
     """
     try:
-        result_dpkg = subprocess.run(['tail', '-n', '10', DPKG_LOG_PATH], stdout=subprocess.PIPE, text=True)
```
Comment (PatrickOrjieh) [inline sensors_script/device_availability_monitor.py:45]: Why did you change it from viewing the last 10 lines to the last 1 line?

Comment (kevashans) [reply sensors_script/device_availability_monitor.py:45]: in the presentation I'm planning to show the error by injecting an error to the log file and then installing something so that the most recent is not an error log. if it checks for the last 10 lines it wont turn green when the most recent log is successful unless i installed the thing 10 times
.

Comment (PatrickOrjieh) [reply sensors_script/device_availability_monitor.py:45]: Okay

**Review by PatrickOrjieh — APPROVED** [review]

**Review by PatrickOrjieh — APPROVED** [review]

**Review by RobertNugent04 — APPROVED** [review]
